packages/flutter/test/material/icon_button_test.dart

```diff
@@ -337,6 +337,37 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('IconButton with explicit splash radius',
+      (WidgetTester tester) async {
+    const double splashRadius = 30.0;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Center(
+            IconButton(
+              icon: const Icon(Icons.android),
+              splashRadius: splashRadius,
+              onPressed: () { /* enable the button */ },
+            ),
```

Comment:
Looks like a compile error here:

```suggestion
            child: IconButton(
              icon: const Icon(Icons.android),
              splashRadius: splashRadius,
              onPressed: () { /* enable the button */ },
            ),
```